docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -1,143 +1,72 @@
 
-[discrete]
-[[synthetics-test-suite-send]]
-=== Send to {stack}
+When you're ready to create a monitor to run your tests, you'll need to set a few configuration options:
 
-After testing locally, you can run the test suite using the Elastic Synthetics integration or {heartbeat}
-depending on <<uptime-set-up-choose,the approach you chose to set up>>.
+* **Give your monitor a name.** Provide a human readable name for the monitor.
+This will appear in {kib} where you can view and manage monitors after they're created.
+* **Set the schedule.** Specify the interval at which your tests will run.
+* **Specify where to find the tests.** This will vary based on the method you use to create a montior.
+* **Set other options as needed.** There are several other options you can set to customize your implementation including params, tags, screenshot options, throttling options, and more.
 
 --
-include::tab-widgets/uptime-monitoring/run-test-suite-widget.asciidoc[]
+include::tab-widgets/uptime-monitoring/run-test-configure-widget.asciidoc[]
 --
 
-[discrete]
-[[synthetics-inline-journey]]
-== Use an inline monitor
-
-An inline monitor is a single journey that you manage individually.
-When writing an inline monitor, the `journey` keyword isn't required, and variables like `page` and `params` will be part of your script's scope.
-
-To create an inline monitor, copy the steps from inside a journey and paste it into {kib} or your {heartbeat} configuration.
-
-Inline monitors can be quick to set up, but can also be more difficult to manage.
-Each inline monitor can contain only _one_ journey, which must be maintained directly in {kib} or in your {heartbeat} configuration file.
-
-If you depend on external packages, have your journeys next to your code repository, or wish to embed and manage more than one journey from a single monitor configuration <<synthetics-test-suite,use a test suite>>.
 
-[discrete]
-[[synthetics-inline-journey-local]]
-=== Test locally
+== Create monitor in {kib}
 
-To test the inline monitor's journey locally, change into your test's directory
-and pipe the journey's contents to the `npx @elastic/synthetics` command.
+After you've decided how you want to configure your monitor, you're ready to create the monitor and start collecting data.
+Push the monitors to {kib} using one of the following methods:
 
-Assume, for example, that your inline monitor includes the following code:
+--
+include::tab-widgets/uptime-monitoring/run-test-suite-widget.asciidoc[]
+--
 
-[source,js]
-----
-step('load homepage', async () => {
-    await page.goto('https://www.elastic.co');
-});
-step('hover over products menu', async () => {
-    await page.hover('css=[data-nav-item=products]');
-});
-----
+// NEED TO INTEGRATE INTO THE NEW CONTENT ON THIS PAGE
 
-To run that journey locally, you can save that code to a file and pipe the file's contents into `@elastic-synthetics`:
+// [discrete]
+// [[synthetics-inline-journey]]
+// == Use an inline monitor
```

Comment:
Still need to integrate this into the above content after getting clarity about how we're framing suites/projects vs inline.